packages/icons/src/library/check.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const saved = (
+	<SVG
+		aria-hidden
```

Comment:
Aren't 3 first props covered by `SVG` component wrapper?

Reply:
Confirmed:
https://github.com/WordPress/gutenberg/blob/46fcca71fb972334710a97d42338f9e4743a148d/packages/primitives/src/svg/index.js#L21-L23

Reply:
Thanks, I fixed your comments in https://github.com/WordPress/gutenberg/pull/19809